server/app/controllers/admin/ProgramAdminManagementController.java

```diff
@@ -85,6 +85,36 @@ public Result update(Http.Request request, long programId) {
     }
   }
 
+  /** Deletes an existing admin email. */
```

Comment:
Nit: can you rephrase this to clarify that it removes a user as a program admin for the specified program? Current phrasing could be read as deleting an email message or resetting an account email, for example.